src/main/java/core/basesyntax/dto/Operations.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.dto;
+
+public enum Operations {
+    b, p, r, s;
+}
```

Comment:
```suggestion
public enum Operation { // it is better to use singular for enum names
    B, P, R, S; // enum constants should be uppercase
}
```

Reply:
then i have to refactor my input.csv and make operation type from "b" to "B" an so on

Reply:
toUpper/LowerCase()? :)